src/deltacode/__init__.py

```diff
@@ -138,6 +141,7 @@ def determine_delta(self):
                 except KeyError:
                     delta = Delta(0, None, old_file)
                     delta.factors.append('removed')
```

Comment:
Now that we have a `status` field for the delta object, we do not need to append that string to our `factors` field.